go/vt/vtorc/logic/topology_recovery.go

```diff
@@ -304,7 +306,13 @@ func postErsCompletion(topologyRecovery *TopologyRecovery, analysisEntry *inst.R
 		_ = AuditTopologyRecovery(topologyRecovery, message)
 		_ = inst.AuditOperation(recoveryName, analysisEntry.AnalyzedInstanceAlias, message)
 		_ = AuditTopologyRecovery(topologyRecovery, fmt.Sprintf("%v: successfully promoted %+v", recoveryName, promotedReplica.InstanceAlias))
-		vtopsExec.RaiseProblem(analysisEntry.AnalyzedInstanceHostname, "orc-dead-tablet")
+
+		// TODO: remove this conditional and 'else' side after vtops-vtorc is used 100%.
+		if strings.HasSuffix(vtopsPath, "vtops-vtorc") {
+			vtopsExec.RaiseProblem(analysisEntry.AnalyzedInstanceAlias, "orc-dead-tablet")
```

Comment:
I made `vtops-vtorc` avoid some extra calls by passing an alias instead of hostname. This means we need this conditional during the rollout